composer.json

```diff
@@ -18,13 +18,13 @@
         "source": "https://github.com/cakephp/elastic-search"
     },
     "require": {
-        "ruflin/elastica": "~3.1",
-        "cakephp/cakephp": "~3.2"
+        "ruflin/elastica": "~3.1"
     },
     "require-dev": {
         "cakephp/cakephp-codesniffer": "dev-master",
         "psr/log": "~1.0",
-        "phpunit/phpunit": "~4.0"
+        "phpunit/phpunit": "~4.0",
+        "cakephp/cakephp": "3.3.*"
```

Comment:
Currently I can't until 3.4.2 is out, since the current version has a problem with phpunit